### PR TITLE
Upgrade Editor and related nuget packages from 15.6.281-preview to 15…

### PR DESCRIPTION
**Pull request by JieCarolHu**
….7.153-preview-g7d0635149a

<details><summary>Upgrade Editor and related nuget packages from 15.6.281-preview to 15.7.153-preview-g7d0635149a</summary>

### Customer scenario
Editor team fixed a bug which is blocking using the Modern ToolTips API, the fix is in 15.7

### Bugs this fixes
https://github.com/dotnet/roslyn/issues/24434

### Workarounds, if any
No

### Risk
Low, this PR only includes nuget package upgrades

### Performance impact
Low, this PR only includes nuget package upgrades

### Is this a regression from a previous update?
NA

### Root cause analysis
NA

### How was the bug found?
NA

### Test documentation updated?
NA

</details>

### CODE REVIEW

**Comment by jasonmalinowski** [comment]
(the change is otherwise great)

**Comment by JieCarolHu** [comment]
retest please

**Comment by JieCarolHu** [comment]
retest windows_build_correctness_prtest please

**Comment by JieCarolHu** [comment]
retest windows_debug_unit32_prtest please

**Comment by JieCarolHu** [comment]
retest windows_determinism_prtest please

**Comment by JieCarolHu** [comment]
retest windows_release_unit32_prtest please

**Comment by JieCarolHu** [comment]
retest windows_release_unit64_prtest please

**Comment by JieCarolHu** [comment]
retest windows_debug_unit64_prtest please

File: src/EditorFeatures/Core.Wpf/Completion/Presentation/CompletionPresenterSession.cs [hunk 1]
```diff
@@ -12,6 +12,7 @@
 using Microsoft.VisualStudio.Text;
 using Microsoft.VisualStudio.Text.Editor;
 using Roslyn.Utilities;
+using CACompletionItem = Microsoft.CodeAnalysis.Completion.CompletionItem;
```
Comment (sharwell) [inline src/EditorFeatures/Core.Wpf/Completion/Presentation/CompletionPresenterSession.cs]: :bulb: Move this inside the namespace declaration and then you can make it this:

```
using CompletionItem = Microsoft.CodeAnalysis.Completion.CompletionItem;
```

The lack of a rename makes this a less intrusive change.

(Applies to other files as well)

File: src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Controller.cs [hunk 2]
```diff
@@ -152,7 +152,7 @@ internal override void OnModelUpdated(Model modelOpt)
 
         private bool StartNewModelComputation(
             CompletionService completionService,
-            CompletionTrigger trigger)
+            CodeAnalysis.Completion.CompletionTrigger trigger)
```
Comment (sharwell) [inline src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Controller.cs]: :bulb: Prefer a using alias declaration to qualification at the use site (applies to other cases as well)

Comment (JieCarolHu) [reply src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Controller.cs]: fixed

File: src/EditorFeatures/Core.Wpf/Completion/Presentation/CompletionPresenterSession.cs (hunk 1 above)
Comment (JieCarolHu) [reply src/EditorFeatures/Core.Wpf/Completion/Presentation/CompletionPresenterSession.cs]: fixed

**Comment by Pilchie** [comment]
Approved for 15.7 P3 - thanks.

**Pull request merged**